src/track/heaptrack.sh.cmake

```diff
@@ -197,7 +205,7 @@ mkfifo $pipe
 # if root is profiling a process for non root
 # give profiled process write access to the pipe
 if [ ! -z "$pid" ]; then
-  pid_user=$(stat -c %U /proc/"$pid")
+  pid_user=$(ls -ld /proc/"$pid" | awk 'NR==1 {print $3}')
```

Comment:
https://www.unix.com/man-page/FreeBSD/1/stat/ - can't we use `stat` on freebsd too, just with a different way to query the `u` user?